curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd31045ce74162adef6c7.md

```diff
@@ -0,0 +1,58 @@
+---
+id: 656cd31045ce74162adef6c7
+title: Task 126
+challengeType: 19
+dashedName: task-126
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Play entire dialogue
+-->
+
+# --description-- 
+
+This is the last task for this dialogue. Listen to the whole conversation again and answer the question below.
+
+# --question--
+
+## --text--
+
+What are Tom and Sophie planning to do?
```

Comment:
As the answers are in English, we could leave the question in English as well, with backticks, thus.